Framework/API/inc/MantidAPI/ISpectrum.h

```diff
@@ -105,7 +105,7 @@ class DLLExport ISpectrum {
     HistogramData::Histogram histogram(std::forward<T>(data)...);
     // Check for the special case EventList, it only accepts histograms without
     // Y and E data.
-    checkHistogram(histogram);
+    checkAndSanitizeHistogram(histogram);
     mutableHistogramRef() = std::move(histogram);
```

Comment:
Yep, I've seen the code that's doing this now. No problems then. I just wanted to check that that situation was being dealt with.